recipe/meta.yaml

```diff
@@ -1,12 +1,13 @@
-{% set version = "3.2.0rc0" %}
+{% set name = "scitools-iris" %}
```

Comment:
This was only done in cf-units because we were re-using the value more than once. Looks like we can't do that here due to the GitHub-PyPI name difference, so would probably be less confusing to remove the variable and hard-code `scitools-iris` into the `url`?

Reply:
I'll see if it works first and refactor it later.